### ISSUE TITLE
bqskit/examples/tutorials directory is outdated

### ISSUE THREAD

**Issue by AbhiDu96** [BQSKit/bqskit#191]
Hello,
I have been trying to run the various synthesis algorithms that BQSkit has to offer on specific unitary gate sets and so I started with the tutorials in the examples directory. But it looks like the modules are not correctly named/imported. So it would be great if the tutorials are updated accordingly.

**Comment by edyounis** [comment]
Yeah, you are right! We should definitely update those. Honestly, all the examples and tutorials should be moved to the [bqskit-tutorials repo](https://github.com/BQSKit/bqskit-tutorial).

**Comment by AbhiDu96** [comment]
Looks like the tutorials have been updated. Thanks a lot for updating them :-)
